server/repository/common.go

```diff
@@ -0,0 +1,6 @@
+package repository
+
+// scannable is used to call .Scan for both sql.Row and sql.Rows
+type scannable interface {
```

Comment:
https://golang.org/pkg/database/sql/#Scanner ?

Reply:
I believe that is the interface used by Rows.Scan, to insert individual values into each argument.

Reply:
Sorry, don't understand you. https://golang.org/pkg/database/sql/#Scanner is absolutely the same as this one.

Reply:
ah. it's actually different. 👍